eva-server/src/main/java/uk/ac/ebi/eva/server/ws/RegionWSServer.java

```diff
@@ -114,4 +114,21 @@ public QueryResponse getVariantsByRegion(@PathVariable("regionId") String region
     public QueryResponse getVariantsByRegion() {
         return setQueryResponse("");
     }
+
+    @RequestMapping(value = "", method = RequestMethod.GET)
```

Comment:
I suggested using the root myself: the root path lists the segments/chromosomes available (I'm not very happy with the terminology but whatever) and those below that level would refer to "resources" in a segment, like variants.